package.json

```diff
@@ -32,6 +32,7 @@
     "babel-plugin-dynamic-import-node": "^2.3.3",
```

Comment:
Do we still need those babel plugins and a Babel config file if we're using ESBuild?

Reply:
> Do we still need those babel plugins and a Babel config file if we're using ESBuild? EDIT: Actually we still need it to run the tests, nevermind.

Ok